yt_dlp/extractor/thisoldhouse.py

```diff
@@ -15,6 +16,11 @@ class ThisOldHouseIE(InfoExtractor):
             'description': 'In the workshop, Tom Silva and Kevin O\'Connor build a storage bench for an entryway.',
             'timestamp': 1442548800,
             'upload_date': '20150918',
+            'duration': 674,
+            'view_count': int,
+            'average_rating': 0,
+            'thumbnail': 'md5:f76278a7a7a63973a98511dc0fb0cf54',
```

Comment:
probly better to use regex here, not md5

Reply:
this is what running the test spit out tbh.
I just copy pasted it so test could pass

Reply:
Fixed